make.sh

```diff
@@ -28,23 +30,40 @@ kill_port_80_usage () {
     fi
 }
 
+find_hostfile () {
+    # try windows first, mac second; add future platforms here.
+    # make sure there aren't any conflicts e.g. the windows bash host may also provide an /etc/hosts file
+    for hosts_candidate in "C:/windows/system32/drivers/etc/hosts" "/etc/hosts"; do
+        echo -n "looking for hostfile at "$hosts_candidate
+        if [ -f "$hosts_candidate" ]; then
+            #file exists, assign hostfile in outer function to current value
+            echo -n "Found host file at ${hosts_candidate}" 
```

Comment:
```suggestion
            echo "Found host file at ${hosts_candidate}" 
```
Removed no newline for same reason: keep output clean.